pkg/cmd/repo/create/create.go

```diff
@@ -369,9 +369,9 @@ func confirmSubmission(repoName string, repoOwner string, isConfirmFlagPassed *b
 
 	promptString := ""
 	if repoOwner != "" {
-		promptString = fmt.Sprintf("This will create '%s/%s' in your current directory. Continue? ", repoOwner, repoName)
+		promptString = fmt.Sprintf("This will create '%s/%s' and add it as remote for your current directory. Continue? ", repoOwner, repoName)
 	} else {
-		promptString = fmt.Sprintf("This will create '%s' in your current directory. Continue? ", repoName)
+		promptString = fmt.Sprintf("This will create '%s' and add it as remote for your current directory. Continue? ", repoName)
```

Comment:
This error message will be accurate for the mode when `repo create` is running within the context of a git repo, but will not be accurate when `repo create` is running _outside_ of a local git repo. In the latter case, there will be an additional prompt to optionally clone the new repository in a subdirectory. See https://github.com/cli/cli/issues/1913

I'm not sure what the solution should be. Please let us discuss this with our team first and get back to you. 🙇